lib/winapi/sync.c

```diff
@@ -9,6 +9,12 @@ VOID InitializeCriticalSection (LPCRITICAL_SECTION lpCriticalSection)
     RtlInitializeCriticalSection(lpCriticalSection);
 }
 
+BOOL InitializeCriticalSectionAndSpinCount (LPCRITICAL_SECTION lpCriticalSection, DWORD dwSpinCount)
+{
+    InitializeCriticalSection(lpCriticalSection);
```

Comment:
We should add a comment that [cites the MSDN](https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-initializecriticalsectionandspincount):

> On single-processor systems, the spin count is ignored and the critical section spin count is set to 0 (zero).

Reply:
Done.